src/App.tsx

```diff
@@ -59,7 +51,9 @@ class App extends React.Component<PropsFromRedux> {
     }
 
     if (type === "column") {
-      const newColumnOrder = Array.from(this.props.allTasks.columnOrder);
+      const newColumnOrder: string[] = Array.from(
```

Comment:
I think `[...this.props.allTasks.columnOrder]` would work her as an alternative to `Array.from(this.props.allTasks.columnOrder)`

Reply:
Didn't change code